Bigtable/tests/System/BackupTests.php

```diff
@@ -0,0 +1,203 @@
+<?php
+/**
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable\Tests\System;
+
+use Google\Cloud\Bigtable\Admin\V2\Backup;
+use Google\Protobuf\Timestamp;
+use Google\ApiCore\ApiException;
+
+/**
+ * @group bigtable
+ * @group bigatableBackup
+ */
+class BackupTests extends BigtableTestCase
+{
+    private static $destinationClusterId;
+    private static $destinationProjectId;
+    private static $backupId;
+    private static $backupName;
+    private static $copyBackupId;
+    private static $expireTime;
+
+    public const DESTINATION_LOCATION_ID = 'us-central1-b';
+
+    public static function set_up_before_class()
```

Comment:
linter seems to complain that you need to use camel casing here:

```suggestion
    public static function setUpBeforeClass(): void
```

Reply:
Oh. Earlier I tried to do that but without the :void thing and was getting error that this function is not compatible with the parent. So didn't knew the solution.

Thanks! Changes incorporated.